.circleci/config.yml

```diff
@@ -8,25 +8,37 @@ commands:
       # Download and cache dependencies
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "project.clj" }}
+            - v2-dependencies-{{ checksum "project.clj" }}
```

Comment:
We don't need to bump the version here. You only need to bump to v2 if you drastically change things in a way where the `project.clj` itself doesn't change.

Reply:
I've changed the command that fetches the dependencies outside of the project.clj file. But you're right, I've also changed the project.clj file.

Reply:
I've force pushed a new version that doesn't change the key.